angular-client/src/components/handling-graphs/long-vs-lat/long-vs-lat.component.html

```diff
@@ -0,0 +1,2 @@
+<info-graph [data]="data" graphContainerId="longVsLatGraph" title="Longitude vs Latitude Acceleration" color="primary">
```

Comment:
longvslat wont work as a line graph, https://apexcharts.com/javascript-chart-demos/radar-charts/ try using this kind of graph